src/lib/types.ts

```diff
@@ -57,6 +57,15 @@ export interface UploadMetadata extends Meta {
 	name: string;
 }
 
+export type AdditionalMetadata = Pick<
+	Metadata,
+	| 'stakingRequests'
+	| 'isBiddable'
+	| 'gridViewByDefault'
+	| 'customDomainHeader'
+	| 'customDomainHeaderValue'
+>;
+
```

Comment:
I am not sure if we still need it 🤔

Reply:
Yeah still for now to default setting grid view. Eventually we can remove this when fixed in backend